servers/physics_3d/godot_body_pair_3d.cpp

```diff
@@ -161,11 +161,10 @@ void GodotBodyPair3D::validate_contacts() {
 	}
 }
 
-// _test_ccd prevents tunneling by slowing down a high velocity body that is about to collide so that next frame it will be at an appropriate location to collide (i.e. slight overlap)
-// Warning: the way velocity is adjusted down to cause a collision means the momentum will be weaker than it should for a bounce!
+// _test_ccd prevents tunneling by moving a high velocity body that is about to collide so that next frame it will be at an appropriate location to collide (i.e. slight overlap)
```

Comment:
```suggestion
// _test_ccd prevents tunneling by moving a high velocity body that is about to collide so that next frame it will be at an appropriate location to collide (i.e. slight overlap).
```